airflow/kubernetes/pod_template_file_examples/dags_in_image_template.yaml

```diff
@@ -36,7 +36,7 @@ spec:
             secretKeyRef:
               name: RELEASE-NAME-fernet-key
               key: fernet-key
-        - name: AIRFLOW__CORE__SQL_ALCHEMY_CONN
```

Comment:
We should also keep the old envvariaable to maintain compatibility with Airflow 1.10. Helm Chart is compatible with Airflow 1.10 and 2.0.

Reply:
Isn't this just an example? or keep old env variable make user friendly ?

Reply:
Oh yes. This file is an example only,  but pod_template_file in /chart/ directory is used by Helm Chart.